lib/processors/lib/jaccard.py

```diff
@@ -0,0 +1,133 @@
+import random
+
+def jaccard_rec(current_answers):
```

Comment:
Why is this in lib? it's obviously a very very very specific function to one of the processors.  Also, what are the `sa*` sets? What is the expected output?